src/Jawa/Event/PlaybackRateChanged.elm

```diff
@@ -0,0 +1,34 @@
+module Jawa.Event.PlaybackRateChanged exposing (PlaybackRateChanged, decoder, encoder)
+
+{-|
+
+@docs PlaybackRateChanged, decoder, encoder
+
+-}
+
+import Json.Decode
+import Json.Encode
+
+
+{-| <https://docs.jwplayer.com/players/reference/playback-events-1#onplaybackratechanged>
+-}
+type alias PlaybackRateChanged =
+    { playbackRate : Float
```

Comment:
The documentation claims this should have a `position` field. I do see that field, but it's always `undefined`.